common/network-common/src/main/java/org/apache/spark/network/buffer/FileSegmentManagedBuffer.java

```diff
@@ -144,10 +145,10 @@ public Object convertToNetty() throws IOException {
 
   @Override
   public String toString() {
-    return Objects.toStringHelper(this)
-      .add("file", file)
-      .add("offset", offset)
-      .add("length", length)
+    return new ToStringBuilder(this, ToStringStyle.SHORT_PREFIX_STYLE)
```

Comment:
The format of the toString is a bit different after this. Before (Guava): `Foo{bar=baz, bing=3}` After (Commons Lang3): `Foo[bar=baz,bing=3]`. I think the exact format is not important for these classes though?